src/IPA.xml

```diff
@@ -0,0 +1,236 @@
+         constitutes the Recipient&apos;s acceptance of this Agreement.</p>
+      <p>Article 1 (Definitions)</p>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>&quot;Digital Font Program&quot; shall mean a computer program containing, or used to render or
+             display fonts.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>&quot;Licensed Program&quot; shall mean a Digital Font Program licensed by the Licensor under
+             this Agreement.</p>
+        </li>
+        <li>
+          <b>3.</b>
+          <p>&quot;Derived Program&quot; shall mean a Digital Font Program created as a result of a
+             modification, addition, deletion, replacement or any other adaptation to or of a part or all
+             of the Licensed Program, and includes a case where a Digital Font Program newly created by
+             retrieving font information from a part or all of the Licensed Program or Embedded Fonts from
+             a Digital Document File with or without modification of the retrieved font information.</p>
+        </li>
+        <li>
+          <b>4.</b>
+          <p>&quot;Digital Content&quot; shall mean products provided to end users in the form of digital
+             data, including video content, motion and/or still pictures, TV programs or other broadcasting
+             content and products consisting of character text, pictures, photographic images, graphic
+             symbols and/or the like.</p>
+        </li>
+        <li>
+          <b>5.</b>
+          <p>&quot;Digital Document File&quot; shall mean a PDF file or other Digital Content created by
+             various software programs in which a part or all of the Licensed Program becomes embedded or
+             contained in the file for the display of the font (&quot;Embedded Fonts&quot;). Embedded Fonts
+             are used only in the display of characters in the particular Digital Document File within
+             which they are embedded, and shall be distinguished from those in any Digital Font Program,
+             which may be used for display of characters outside that particular Digital Document File.</p>
+        </li>
+        <li>
+          <b>6.</b>
+          <p>&quot;Computer&quot; shall include a server in this Agreement.</p>
+        </li>
+        <li>
+          <b>7.</b>
+          <p>&quot;Reproduction and Other Exploitation&quot; shall mean reproduction, transfer, distribution,
+             lease, public transmission, presentation, exhibition, adaptation and any other
+             exploitation.</p>
+        </li>
+        <li>
+          <b>8.</b>
+          <p>&quot;Recipient&quot; shall mean anyone who receives the Licensed Program under this Agreement,
+             including one that receives the Licensed Program from a Recipient.</p>
+          <p>Article 2 (Grant of License)</p>
```

Comment:
List hierarchy has gone wrong here. These "article" lines need to be outdented. Pull request submitted.